src/main/java/hillel/spring/DoctorRestAPI/dto/DoctorDtoConverter.java

```diff
@@ -0,0 +1,17 @@
+package hillel.spring.DoctorRestAPI.dto;
+
+import hillel.spring.DoctorRestAPI.Doctor;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+import java.util.List;
+
+@Mapper
+public interface DoctorDtoConverter {
+    @Mapping(target = "id", ignore = true)
+    Doctor toModel(DoctorInputDto dto);
+
+    DoctorOutputDto toModel(Doctor doc);
+
+    List<DoctorOutputDto> toModel(List<Doctor> docs);
```

Comment:
If you have `Model` -> `DTO` function it is not the bests idea to have yet another `List<Model>`->`List<DTO>` one. because List (through stream) is a Functor and you always can apply `dtos.stream().map(::toModel).collect(toList())` and achieve the same functionality. 
Also, this approach works with other collections as Sets.